pkg/remote/remote.go

```diff
@@ -87,7 +86,9 @@ WORKDIR /okteto/src
 {{range $key, $val := .OktetoBuildEnvVars }}
 ENV {{$key}} {{$val}}
 {{end}}
-
+{{range $key, $val := .OktetoCommandSpecificEnvVars }}
+ARG {{$key}} {{$val}}
```

Comment:
Since the image won't be published to a registry, I defined the variables as `ARG` in the Dockerfile instead of `ENV`. This approach makes sense because the variables won’t be accessible in the container once built, and since we aren’t pushing the image, it avoids unnecessary exposure. By using `ARG`, we can pass empty environment variables during the `okteto test` command without worrying if they’re set to empty values.